include/IComponent.hpp

```diff
@@ -18,6 +19,8 @@ namespace arcade
     protected:
     private:
     };
+    /* There are several components inheriting from IComponent and used by game and graphical libraries. To create and use libraries you must handle all components. */
+    /* > IT IS NOT POSSIBLE TO CREATE NEW COMPONENTS DUE TO THE NECESSITY FOR ALL LIBRARIES TO KNOW THE SAME COMPONENTS */
```

Comment:
You can add new components if they're used only for the library where they are defined.